distribution/docs/src/main/resources/_contents/_sources/csw-connected-source-contents.adoc

```diff
@@ -0,0 +1,28 @@
+
+==== CSW Connected Source
+
+The CSW connected source supports the ability to search collections of descriptive information (metadata) for data, services, and related information objects.
+
+===== Installing the CSW Connected Source
+
+The CSW Connected Source is | is not installed by default with a standard installation in the ${ddf-spatial} application.
```

Comment:
It is not. I think it might even be removed, someone (I think @kcwire) said it was